tests/runtimes/test_run.py

```diff
@@ -13,6 +13,7 @@ def test_new_function_from_runtime():
             "tag": "",
             "hash": "7b3064c6b334535a5d949ebe9cfc61a094f98c78",
             "updated": "2020-10-21T22:40:35.042132+00:00",
+            "credentials": {"access_key": "some-access-key"},
         },
         "spec": {
             "command": "spark-submit",
```

Comment:
I suggest adding another test verifying the `submit_job` API - it could be part of the `test_submit_job_auto_mount` test that I've recently added (or a specific test just for this) - the same mocks should be used etc. Make the API call, and verify that the new env. variables are set properly for the pod created.
Hopefully at some point in the future I'll add a test for the `ingest` API, and then these changes can be verified there as well.

Reply:
Added